app/views/static_pages/about.html.haml

```diff
@@ -38,10 +38,10 @@
 %h3#leadership Leadership
 
 .leader
-  =image_tag "mary_becica.jpg"
-  %h4 Mary Becica, Board Member, Executive Director
+  =image_tag "golden.jpg"
+  %h4 Alexsarah "Golden" Collier, Board Member, Executive Director
 
-  %p Mary Becica chairs the space committee at Double Union and works on designing and building the physical space. During the day Mary is a Director of Product Management at AppDirect, where she meets with a lot of people and thinks about enterprise software management. At night she works on data visualization projects, thinks about cryptocurrency futurism, and reads a lot. You can find her at @mbecica.
+  %p Golden is a Learning Specialist at an SF elementary school by day. He is also a teaching member of Girl Army, an Oakland-based collective providing strengths-based sliding scale self defense courses for women, gender non-conforming/non-binary, and trans folks. Also, Golden is a member of Sula Collective, maker @ <a href="kidsizzle.etsy.com">kidsizzle.etsy.com</a>, apprentice-level beekeeper, motorcyclist, digital media activist with Womanist Trilliance and Combahee Radio, former world record holder, and metalsmith who enjoys dreamy hikes, practicing mindfulness, playing the viola/banjo, singing, the ocean, and libraries.
```

Comment:
@mathildemouw The link to Golden's Etsy page is trying to go to http://doubleunion.org/kidsizzle.etsy.com. What do you think about updating this to use the rails link_to helper and make it `target: '_blank'`?